### DISCUSSION TITLE
Is it possible to get the example body schema as string or dict?

### DISCUSSION THREAD

**Discussion by JavierSanchezCastro** [fastapi/fastapi#9593] [Questions]
### First Check

- [x] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
No example code
```

### Description

![imagen](https://github.com/tiangolo/fastapi/assets/72013291/9793014a-e1a4-42a1-b658-2c9bb26e08c1)

Is there any way to get the example schema generated in /docs as dict or string from a BaseModel class?

I tried to use this:

```python
print(ClassName.schema())
print(ClassName.schema_json())
```

but it's not working as expected

Thanks!

### Operating System

Linux

### FastAPI Version

0.95.2

### Python Version

Python 3.10.6

**Comment by SahandJ** [comment]
The `.schema()` should return a dictionary with all the required information. You can write a simple helper function to parse this into however you want:

```python
def get_basic_schema(model: BaseModel) -> dict[str, str]:
    return {
        key: value['type']
        for key, value in model.schema()['properties'].items()
      }
```

For more information about the `.schema()` function see: https://docs.pydantic.dev/latest/usage/schema/
